src/osp/UserInputHandler.cpp

```diff
@@ -257,7 +264,7 @@ ButtonControlHandle UserInputHandler::config_get(std::string const& name)
 
     if (cfgIt == m_controlConfigs.end())
     {
-        // Config not found!
+        // Config not found, crashes after this, so needs to be fixed
```

Comment:
If it's going to crash if it's not found, then just explicitly crash the program, don't rely on it crashing on it's own.

do something like `throw std::exception("some explaination");`